src/ziggurat/config.clj

```diff
@@ -39,7 +39,8 @@
                                 :http-server          {:middlewares  {:swagger {:enabled false}}
                                                        :port         8080
                                                        :thread-count 100}
-                                :new-relic            {:report-errors false}}})
+                                :new-relic            {:report-errors false}
+                                :log-format           "text"}})
```

Comment:
Default should be json.

Reply:
initially it would be better if the default were `text`. This will allow users to make a gradual  shift to JSON logging